src/watch/index.ts

```diff
@@ -139,6 +139,10 @@ export class Task {
 		});
 		this.inputOptions = inputOptions;
 
+		if (config && config.cache) {
```

Comment:
Is it possible the config is not an object? Otherwise the check can be shortened.

Reply:
you are right,  it my habitual behavior.